feat: playground (#2084)
amannn/next-intl: examples/example-app-router-patterns/src/app/[locale]/translations/client-components/content.mdx

> - `useTranslations` works the same in Client Components, so interactive UI can read messages too.

Reviewer: ```suggestion
- `useTranslations` can be called in Client Components to incorporate client-side state.
```

> - ICU arguments like `{name}` are resolved in the browser as state changes — try typing below.

Reviewer: Coming from https://github.com/amannn/next-intl/pull/2084#discussion_r3324274771: Yeah, I'd switch to inline labels here. Otherwise the locale switcher will appear to be non-working if we have markdown that is not translated. Using inline labels with `t` should be easier to maintain (esp. with `useExtracted`).

We can also avoid the `@next/mdx` dependency then I think if we use `codehike` directly on a page and pass code samples like below directly as multiline strings. This will also enable Turbopack, since the way `next/mdx` is used currently causes a switch to webpack.

> - ICU arguments like `{name}` are resolved in the browser as state changes — try typing below.

Reviewer: I think the "try typing below." is not necessary.

There's this phrase I like which I think you already did very well with the input field: Show, don't tell.

> ```tsx app/greet.tsx

Reviewer: I think `app/greet.tsx` doesn't add anything here. We could however use a generic "Code" label, related to "Output" below.

'use client';
import {useState} from 'react';
import {useTranslations} from 'next-intl';

export function Greet() {
  const t = useTranslations('ClientComponentsPage');
  const [name, setName] = useState('Frodo');
  return (
    <>
      <input value={name} onChange={(e) => setName(e.target.value)} />

>       // !mark

Reviewer: This is rendered on the page:

<img width="553" height="120" alt="Image" src="https://github.com/user-attachments/assets/6aa2ad58-4f16-4576-bad6-2b36038704b9" />

I guess the issue is that `//` comments don't work in JSX (only /* … */)
